docker_requirements.txt

```diff
@@ -0,0 +1,19 @@
+Flask
```

Comment:
I think I see what you're trying to do here... install everything but GDAL since it's already installed in the base container?

But the down side is that this is almost 100% duplication of `requirements.txt`.

Maybe we should hold off on this. I already have [a branch](https://github.com/pacificclimate/climate-explorer-backend/tree/upgrade-geo) going that upgrades everything to GDAL 3.x and uses the pcic/geospatial-python:gdal3 image. It should be ready shortly...

Reply:
Sounds good!